tests/executable.rs

```diff
@@ -0,0 +1,61 @@
+use anyhow::{Context, Result};
+use assert_cmd::cargo::cargo_bin;
+use std::fs;
+use std::{thread, time};
+use tempfile::tempdir;
+use tokio::process::Command;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn relays_signals() -> Result<()> {
+    let dir = tempdir()?;
+    let mut cmd = Command::new(cargo_bin("comment"));
+    let outpath = dir.path().join("signal");
+    let child = cmd.args([
+        "--",
+        "./signal.sh",
+        outpath
+            .as_os_str()
```

Comment:
```suggestion
```

`PathBuf` it seems also has a `to_str` on it with the same signature as `OsStr`'s.